demos/crossroad_camera_demo/crossroad_camera_demo.hpp

```diff
@@ -10,7 +10,8 @@
 #include <gflags/gflags.h>
 
 static const char help_message[] = "Print a usage message.";
-static const char video_message[] = "Required. Path to a video or image file. Default value is \"cam\" to work with camera.";
+static const char input_message[] = "Required. Input to process.";
```

Comment:
Since this is a very generic description, I think it would be useful to add a section to the documentation to explain what's supported as an input.

Reply:
If we are going to describe what input is we better keep it in help message. I've included that into a macro.